### PR TITLE
README updated with new namespace, poetry cmds

### PR DESCRIPTION
**Pull request by stratofax**
README.md now includes:

- Version requirement: Python 3.9 or greater
- The new VersaDM naming convention
- A brief section on using `poetry` for package management
- Updated Docker commands for `versadm`

I also included the `poetry.lock` file I generated when I executed `poetry install`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
 
-## Set up venv
+### Using Poetry
+
+We use [Poetry for Python dependency management and packaging](https://python-poetry.org/) for continuous development and integration work. If you're interested in working with the code in this repo, follow the instructions to [install poetry on your development workstation](https://python-poetry.org/docs/#installation).
+
+Then, you can [set up your development environment using the poetry.lock file](https://python-poetry.org/docs/basic-usage/#installing-with-poetrylock). This consists of entering the following command in the project root directory:
+
+```bash
+poetry install
```
Comment (ion-oset) [inline README.md:24]: Add a note about using `poetry shell` to activate the environment `install` creates.

Comment (stratofax) [reply README.md:24]: Updated as suggested, thanks Ion!

**Review by alanyst — APPROVED** [review]

**Review by alanyst — CHANGES_REQUESTED** [review]
On reflection, I think we should omit `poetry.lock` from the repo and add it to .gitignore, since it can be generated on demand and is more like a build artifact than a source file.

**Comment by stratofax** [comment]
Agreed, so I removed `poetry.lock` from this PR.

**Comment by stratofax** [comment]
Agree with Alan, `poetry.lock` removed from this PR.

**Review by alanyst — APPROVED** [review]

**Pull request merged**